scripts/zones/Balgas_Dais/mobs/Wyrm.lua

```diff
@@ -2,19 +2,29 @@
 -- Area: Balga's Dais
 --  Mob: Wyrm
 -- KSNM: Early Bird Catches the Wyrm
+-- For future reference: Trusts are not allowed in this fight 
```

Comment:
@UynGH Did you try to summon trusts in the BCNM? I saw Zach did, but he wasn't the leader 😆 .

Reply:
Hello! Just checked:
![1](https://user-images.githubusercontent.com/40763842/96615452-92940000-1301-11eb-829f-a7697681c48b.jpg)

Reply:
As a reminder to myself:
https://github.com/project-topaz/topaz/blob/release/src/map/battlefield.h#L34

I need to add RULES_NO_TRUSTS